API/things-api/app.js

```diff
@@ -177,64 +177,194 @@ app.get('/shoppinglist', function(req, res){
 
 
 /****************************************************
-* /path     /stats/:id
-* /params
-*
+* /path     /stats/:name
+* /params   :name - item_naem of what we are looking for
 *
 * /brief    Route to pull satistics for an item
-*           should probably be renamed to be more specific
 *
 * /author   Andrew McCann
 * /date     2/3/2017
 ****************************************************/
-app.get('/stats/:id', function(err, client, done) {
+app.get('/stats/:name', function(req, res) {
```

Comment:
I think that :id was correct here, the front end will be submitting requests by item id not by the name